### PR TITLE
Dockerizing hangoff

### PR DESCRIPTION
**Pull request by M-Wittner**
Hi Benny!

I've Dockerized the app for both dev and prod uses.
Included a docker.md file with instructions.

LMK if that works for you :)

### CODE REVIEW

File: .example.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SERVER_PORT=5545
```
Comment (benny-n) [inline .example.env]: This is the server's dotenv file, so it should probably be moved to `/hangoff-server` directory

File: docker.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Development In Docker
+
```
Comment (benny-n) [inline docker.md]: These instructions are great!
However, I think it'll be best if we merge this content with the main README.md file

**Review by benny-n — CHANGES_REQUESTED** [review]
Great job 👍

File: docker.md (hunk 2 above)
Comment (M-Wittner) [reply docker.md]: Added docker instruction to main readme file.

File: .example.env (hunk 1 above)
Comment (M-Wittner) [reply .example.env]: Of course...

**Review by benny-n — APPROVED** [review]

**Pull request merged**